<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>350</width>
    <height>279</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>350</width>
    <height>279</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>350</width>
    <height>279</height>
   </size>
  </property>
  <property name="palette">
   <palette>
    <active>
     <colorrole role="WindowText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Button">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>248</red>
        <green>248</green>
        <blue>248</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Light">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Midlight">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>251</red>
        <green>251</green>
        <blue>251</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Dark">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>124</red>
        <green>124</green>
        <blue>124</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Mid">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>165</red>
        <green>165</green>
        <blue>165</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Text">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="BrightText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ButtonText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>248</red>
        <green>248</green>
        <blue>248</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Shadow">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="AlternateBase">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>251</red>
        <green>251</green>
        <blue>251</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ToolTipBase">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>220</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ToolTipText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
    </active>
    <inactive>
     <colorrole role="WindowText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Button">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>248</red>
        <green>248</green>
        <blue>248</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Light">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Midlight">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>251</red>
        <green>251</green>
        <blue>251</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Dark">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>124</red>
        <green>124</green>
        <blue>124</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Mid">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>165</red>
        <green>165</green>
        <blue>165</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Text">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="BrightText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ButtonText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>248</red>
        <green>248</green>
        <blue>248</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Shadow">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="AlternateBase">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>251</red>
        <green>251</green>
        <blue>251</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ToolTipBase">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>220</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ToolTipText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
    </inactive>
    <disabled>
     <colorrole role="WindowText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>124</red>
        <green>124</green>
        <blue>124</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Button">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>248</red>
        <green>248</green>
        <blue>248</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Light">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Midlight">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>251</red>
        <green>251</green>
        <blue>251</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Dark">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>124</red>
        <green>124</green>
        <blue>124</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Mid">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>165</red>
        <green>165</green>
        <blue>165</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Text">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>124</red>
        <green>124</green>
        <blue>124</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="BrightText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ButtonText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>124</red>
        <green>124</green>
        <blue>124</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>248</red>
        <green>248</green>
        <blue>248</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>248</red>
        <green>248</green>
        <blue>248</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Shadow">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="AlternateBase">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>248</red>
        <green>248</green>
        <blue>248</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ToolTipBase">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>220</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ToolTipText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
    </disabled>
   </palette>
  </property>
  <property name="cursor">
   <cursorShape>ArrowCursor</cursorShape>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::DefaultContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>impRef</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../impRef_frame01.qrc">
    <normaloff>:/test/window_icon.png</normaloff>:/test/window_icon.png</iconset>
  </property>
  <property name="windowOpacity">
   <double>2.000000000000000</double>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="toolButtonStyle">
   <enum>Qt::ToolButtonIconOnly</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="BG">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>0</y>
      <width>371</width>
      <height>261</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>371</width>
      <height>261</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>371</width>
      <height>261</height>
     </size>
    </property>
    <property name="sizeIncrement">
     <size>
      <width>371</width>
      <height>261</height>
     </size>
    </property>
    <property name="baseSize">
     <size>
      <width>371</width>
      <height>261</height>
     </size>
    </property>
    <property name="palette">
     <palette>
      <active>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="100">
          <red>0</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Text">
        <brush brushstyle="SolidPattern">
         <color alpha="100">
          <red>0</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="ButtonText">
        <brush brushstyle="SolidPattern">
         <color alpha="100">
          <red>0</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
      </active>
      <inactive>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="100">
          <red>0</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Text">
        <brush brushstyle="SolidPattern">
         <color alpha="100">
          <red>0</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="ButtonText">
        <brush brushstyle="SolidPattern">
         <color alpha="100">
          <red>0</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
      </inactive>
      <disabled>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="100">
          <red>0</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Text">
        <brush brushstyle="SolidPattern">
         <color alpha="100">
          <red>0</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="ButtonText">
        <brush brushstyle="SolidPattern">
         <color alpha="100">
          <red>0</red>
          <green>0</green>
          <blue>0</blue>
         </color>
        </brush>
       </colorrole>
      </disabled>
     </palette>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">.QFrame{
background-image : url(:/test/BG.png);
background-repeat: no-repeat;
background-position: center;
color: rgba(0, 0, 0, 100);
}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="ok_button">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>210</y>
       <width>61</width>
       <height>41</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>61</width>
       <height>41</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>61</width>
       <height>41</height>
      </size>
     </property>
     <property name="cursor">
      <cursorShape>ClosedHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../impRef_frame01.qrc">
       <normaloff>:/test/ok_button.png</normaloff>:/test/ok_button.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>60</width>
       <height>36</height>
      </size>
     </property>
     <property name="default">
      <bool>false</bool>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QCommandLinkButton" name="Q_button">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>0</y>
       <width>31</width>
       <height>41</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>ClosedHandCursor</cursorShape>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../impRef_frame01.qrc">
       <normaloff>:/test/Q_button.png</normaloff>:/test/Q_button.png</iconset>
     </property>
    </widget>
    <widget class="QCommandLinkButton" name="Q_button_3">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>0</y>
       <width>31</width>
       <height>41</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>UpArrowCursor</cursorShape>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../impRef_frame01.qrc">
       <normaloff>:/test/owo_button.png</normaloff>
       <disabledoff>:/test/owo_button.png</disabledoff>:/test/owo_button.png</iconset>
     </property>
    </widget>
    <widget class="QCommandLinkButton" name="Q_button_4">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>0</y>
       <width>31</width>
       <height>41</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>ArrowCursor</cursorShape>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../impRef_frame01.qrc">
       <normaloff>:/test/heart_button.png</normaloff>
       <disabledoff>:/test/heart_button.png</disabledoff>:/test/heart_button.png</iconset>
     </property>
    </widget>
    <widget class="QWidget" name="creat1_tex" native="true">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>20</y>
       <width>281</width>
       <height>81</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">image: url(:/test/create_text.png);</string>
     </property>
    </widget>
    <widget class="QPushButton" name="createCam_button">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>90</y>
       <width>141</width>
       <height>111</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>ClosedHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgba(0, 0, 0, 0);</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../impRef_frame01.qrc">
       <normaloff>:/test/create_button.png</normaloff>:/test/create_button.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>133</width>
       <height>97</height>
      </size>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="name_box">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>200</y>
       <width>141</width>
       <height>20</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">image: url(:/test/rename_box.png);</string>
     </property>
     <property name="inputMethodHints">
      <set>Qt::ImhNone</set>
     </property>
     <property name="frame">
      <bool>true</bool>
     </property>
     <property name="cursorPosition">
      <number>0</number>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="clearButtonEnabled">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="selectCam_button">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>90</y>
       <width>141</width>
       <height>111</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>ClosedHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgba(0, 0, 0, 100);</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../impRef_frame01.qrc">
       <normaloff>:/test/selectCam_button.png</normaloff>:/test/selectCam_button.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>122</width>
       <height>114</height>
      </size>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="line_BG">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>90</y>
       <width>16</width>
       <height>111</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>ArrowCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../impRef_frame01.qrc">
       <normaloff>:/test/line_BG.png</normaloff>
       <normalon>:/test/line_BG.png</normalon>
       <disabledoff>:/test/line_BG.png</disabledoff>:/test/line_BG.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>6</width>
       <height>142</height>
      </size>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>350</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="../impRef_frame01.qrc"/>
 </resources>
 <connections/>
</ui>
